### ISSUE TITLE
High CPU usage when completely idle

### ISSUE THREAD

**Issue by jkugler** [sindresorhus/caprine#1669]
```
$ uname -a
Linux hosanna 5.11.0-7620-generic #21~1624379747~20.04~3abeff8~dev-Ubuntu SMP Tue Jun 22 22:08:51  x86_64 x86_64 x86_64 GNU/Linux
$ dpkg -l caprine
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version      Architecture Description
+++-==============-============-============-======================================
ii  caprine        2.51.2       amd64        Elegant Facebook Messenger desktop 
$ kded5 --version
kded5 5.68.0
```
Even when completely idle, doing nothing, not receiving messgaes, Carprine will sit at 15% CPU usage. I would expect an idle application to use *zero* CPU, or at most, 1% or 2% while monitoring for new messages.

**Comment by retsimx** [comment]
Same for me.

```❯ uname -a
Linux LewisLaptop 5.10.52-1-lts #1 SMP Tue, 20 Jul 2021 16:46:09 +0000 x86_64 GNU/Linux
❯ pacman -Qe caprine
caprine 2.53.0-1
❯ kded5 --version
kded5 5.84.0```

Consistently using 10-15% while idle and even while minimised. No videos, gifs etc are playing.

**Comment by Fiaegel** [comment]
On linux, this program uses 50% of a cpu's core and an absurd amount of ram for what is essentially a toolbar-less chromium window. Something's not working as intended for sure lol

**Comment by jkugler** [comment]
@lefterisgar Can you explain why this was closed as complete? This issue still exists. Caprine still uses way to much CPU at idle.

**Comment by lefterisgar** [comment]
Yes, because Electron has been updated and it should be improved in the next release.

**Comment by lefterisgar** [comment]
If not, reopen the issue.

**Comment by jkugler** [comment]
Ah, gotcha. Sounds good! Do you know when the release will be? I'll look for it then!

**Comment by lefterisgar** [comment]
Unfortunately I'm not the one responsible for the release. In the meantime, try [my fork](https://github.com/lefterisgar/caprine/releases) to see if the issue is resolved.

**Comment by jkugler** [comment]
I just updated to 2.56.0, which uses the new Electron release, and Caprine is still using 15-20% CPU when idle, and spiking higher than that more often than the older version.

**Comment by lefterisgar** [comment]
Do you want to reopen the issue?

**Comment by lefterisgar** [comment]
By the way, my i3 10th gen doesn't seem to go more than 3% while idling with Caprine open of course.

What is your OS?

**Comment by jkugler** [comment]
Yes, please reopen the issue.

This is a Intel(R) Core(TM) i7-4710MQ (don't know the generation). Linux, Ubuntu 20.04. 

If a program is idling, and maybe polling a bit to check for messages (I don't know how messenger works) it shouldn't be taking more then 1 or 2% CPU. Using 15% CPU indicates there is some serious work going on. I don't know what it would be doing with no data or screen updates happening.

**Comment by lefterisgar** [comment]
> Yes, please reopen the issue.
> 
> This is a Intel(R) Core(TM) i7-4710MQ (don't know the generation). Linux, Ubuntu 20.04.
> 
> If a program is idling, and maybe polling a bit to check for messages (I don't know how messenger works) it shouldn't be taking more then 1 or 2% CPU. Using 15% CPU indicates there is some serious work going on. I don't know what it would be doing with no data or screen updates happening.

The first letter after i7 is the generation so yours is 4th gen. I will leave this open so someone who knows better can investigate but I think it's definitely Electron related.

**Comment by lefterisgar** [comment]
@jkugler Would you mind if I closed it in favor of #1882 which is more recent and a lot of people will read it?

**Comment by jkugler** [comment]
That will work!
